task_11/css/style.css

```diff
@@ -0,0 +1,982 @@
+
+}
+
+.summary-table th {
+  color: #6e768f;
+  letter-spacing: 1px;
+  text-transform: uppercase;
+  font-weight: 700;
+  text-align: left;
+  padding: 5px 20px;
+}
+
+/*Summary buttons*/
+
+.summary-buttons {
+  margin: 20px 0;
+  margin-right: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 25%;
+}
+
+.summary-buttons button {
+  position: relative;
+  width: 200px;
+  height: 65px;
+  margin: 10px auto;
+  font-size: 15px;
+  font-weight: 700;
+  text-transform: uppercase;
+  text-align: center;
+  border-radius: 5px;
+}
+
+.summary-buttons button[name="submit"] {
+  color: #fff;
+  background-color: #5fcac6;
+  border: 2px solid #54aca9;
+}
+
+.summary-buttons button[name="quote"] {
+  color: #a5abc5;
+  background-color: #f7f7fa;
+  border: 2px solid #d7d8e0;
+}
+
+/*Summary help*/
+
+.summary-help {
+  display: flex;
+  flex-direction: column;
+  width: 25%;
+  margin: 40px 20px 10px 40px;
+}
+.summary-help a {
+  display: inline-block;
+  padding-top: 10px;
+  padding-right: 20px;
+  text-decoration: none;
+  color: #6e768f;
+  font-weight: 700;
+}
+
+.summary-help p {
+  display: inline-block;
+  margin-left: 45px;
+  line-height: 20px;
+}
+
+/*Form*/
+
+.form {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  counter-reset: section;
+}
+
+/*Counters*/
+
+h2::before {
+  position: absolute;
+  top: -15px;
+  left: -80px;
+  counter-increment: section;
+  content: counter(section);
+  display: inline-block;
+  width: 20px;
+  height: 20px;
+  padding: 12px;
+  text-align: center;
+  border-radius: 50%;
+  border: 2px solid #6e768f;
+}
+
+/*Document*/
+
+.order-form {
+  margin: 60px;
+}
+
+.document-container {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  position: relative;
+  margin: 30px 20px;
+}
+
+.doc-wrapper {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.radio-wrapper img {
+  width: 120px;
+}
+
+.h1 {
+  display: inline-block;
+  font-size: 25px;
+  text-align: center;
+  letter-spacing: 3px;
+  margin-bottom: 50px;
+  padding: 50px 5px;
+  padding-bottom: 15px;
+  border-bottom: 1px solid #d7d8e0;
+}
+
+.radio-wrapper {
+  display: block;
+  position: relative;
+  width: 300px;
+  height: 180px;
+  font-family: "Proxima Nova Bold";
+  font-size: 20px;
+  margin: 10px 10px;
+  padding: 24px 7px;
+  text-align: center;
+  background-color: #f7f7fa;
+  border: 1px solid #d7d9e0;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+.radio-wrapper:hover {
+  opacity: 0.5;
+}
+
+.visually-hidden {
+  position: absolute;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
+  height: 1px;
+  width: 1px;
+  margin: -1px;
+  padding: 0;
+  border: 0;
+}
+
+input[name="document"]:checked + .radio-wrapper {
+  display: block;
+  padding: 17px 0;
+  border: 8px solid #76a7e1;
+  border-radius: 5px;
+}
+
+input[name="document"]:checked + .radio-wrapper:after {
+  font-family: "Font Awesome 5 Free";
+  content: "\f058";
```

Comment:
take a look at this solution:
https://codepen.io/lempiy/pen/aEQrxg

Reply:
Thanks for the hint! Works great!